support/gin/gin.go

```diff
@@ -0,0 +1,49 @@
+package gin
+
+import (
+	"net/http"
+
+	"github.com/davidebianchi/gswagger/apirouter"
+	
+	"github.com/gin-gonic/gin"
+)
+
+// HandlerFunc is the http type handler used by Gin
+type HandlerFunc gin.HandlerFunc
+type Route = gin.IRoutes
+
+type ginRouter struct {
+	router *gin.Engine
+}
+
+func (r ginRouter) AddRoute(method string, path string, handler HandlerFunc) Route {
+	switch method {
+	case http.MethodGet:
+		return r.router.GET(path, gin.HandlerFunc(handler))
+	case http.MethodPost:
+		return r.router.POST(path, gin.HandlerFunc(handler))
+	case http.MethodPut:
+		return r.router.PUT(path, gin.HandlerFunc(handler))
+	case http.MethodDelete:
+		return r.router.DELETE(path, gin.HandlerFunc(handler))
+	default:
+		return r.router.Any(path, gin.HandlerFunc(handler))
+	}
+}
+
+func (r ginRouter) SwaggerHandler(contentType string, blob []byte) HandlerFunc {
+	return func(c *gin.Context) {
+		c.Header("Content-Type", contentType)
+		c.Data(http.StatusOK, contentType, blob)
+	}
+}
+
+func (r ginRouter) TransformPathToOasPath(path string) string {
+	return path
```

Comment:
Gin seems to support path params [with colon](https://gin-gonic.com/docs/examples/param-in-path/), is it true? It should be converted to oas spec with [this](https://github.com/davidebianchi/gswagger/blob/main/apirouter/transformpath.go#L7). Can you see an example usage in fiber support.